testing/events.go

```diff
@@ -135,6 +135,20 @@ func ParseAckFromEvents(events []abci.Event) ([]byte, error) {
 	return nil, fmt.Errorf("acknowledgement event attribute not found")
 }
 
+// ParseProposalIDFromEvents parses events emitted from MsgSubmitProposal and returns proposalID
+func ParseProposalIDFromEvents(events []abci.Event) (uint64, error) {
+	for _, event := range events {
+		for _, attribute := range event.Attributes {
+			if attribute.Key == "proposal_id" {
+				proposalID, err := strconv.ParseUint(attribute.Value, 10, 64)
```

Comment:
nit: I think you could return directly the call to `strconv.ParseUint` instead of using the vars

Reply:
thank @damiannolan, updated!